core/src/impl/Cabana_PerformanceTraits.hpp

```diff
@@ -83,6 +83,16 @@ class PerformanceTraits<Kokkos::Experimental::HIP>
 };
 #endif
 
+//---------------------------------------------------------------------------//
+#if defined( KOKKOS_ENABLE_OPENMPTARGET )
+template <>
+class PerformanceTraits<Kokkos::Experimental::OpenMPTarget>
+{
+  public:
+    static constexpr int vector_length = 16; // FIXME_OPENMPTARGET
```

Comment:
@dalg24 Will the Kokkos interface have something like the warp size parameter based on the device this was compiled for?